From d15ee692af42163846be7517372f3ee098cdaf18 Mon Sep 17 00:00:00 2001
From: Beniamin Sandu <beniaminsandu@gmail.com>
Date: Thu, 15 Jun 2023 17:02:58 +0000
Subject: [PATCH] remove unneeded headers

From a quick look, these seem to not be needed and having them
breaks the build with mbedtls 3.x. Without them it builds fine
on both 2.x and 3.x versions.

Upstream-Status: Inactive-Upstream [Upstream does not take patches]
Signed-off-by: Beniamin Sandu <beniaminsandu@gmail.com>
---
 mDNSPosix/mbedtls.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/mDNSPosix/mbedtls.c b/mDNSPosix/mbedtls.c
index a73681b6d528..ab8f8c772c00 100644
--- a/mDNSPosix/mbedtls.c
+++ b/mDNSPosix/mbedtls.c
@@ -38,10 +38,8 @@
 #include <mbedtls/sha256.h>
 #include <mbedtls/base64.h>
 
-#include <mbedtls/certs.h>
 #include <mbedtls/x509.h>
 #include <mbedtls/ssl.h>
-#include <mbedtls/config.h>
 
 // Posix TLS server context
 struct TLSContext_struct {
